packages/twenty-server/src/modules/workflow/common/types/workflow-step.type.ts

```diff
@@ -1,9 +1,19 @@
-import { WorkflowCodeSettings } from 'src/modules/workflow/common/types/workflow-settings.type';
-
 export enum WorkflowStepType {
   CODE_ACTION = 'CODE_ACTION',
+  SEND_EMAIL_ACTION = 'SEND_EMAIL_ACTION',
 }
 
+type BaseWorkflowSettings = {
+  errorHandlingOptions: {
```

Comment:
it seems wrong to have this type in workflow-step.type file

Reply:
ok, i will add back the `workflow-setttings.type.ts` file